modules/nodes-scheduler-node-selectors-pod.adoc

```diff
@@ -171,11 +178,11 @@ spec:
         node-role.kubernetes.io/worker: ''
         type: user-node <1>
 ----
-<1> Add the desired node selector.
-+
-* For a new pod, you can add the selector to the pod specification directly:
+<1> Add the node selector.
+
+* To add a node selector to a specific pod, add the selector to the `Pod` object directly:
```

Comment:
I'd add the steps to modify the replicaset and pod.

Reply:
You cannot add a node selector to an existing pod directly on the controlling object.